wgpu-core/src/device/mod.rs

```diff
@@ -5437,41 +5512,31 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             }
             resource::BufferMapState::Active { ptr, ref range, .. } => {
                 if offset < range.start {
-                    return Err(resource::BufferAccessError::OutOfBoundsUnderrun {
+                    return Err(BufferAccessError::OutOfBoundsUnderrun {
```

Comment:
This is fine for this PR, but for the future, it helps me review more quickly and accurately if mechanical changes like this are separated out from substantive ones. Putting it in either a separate PR or just a separate commit within the PR is fine.